taskt/Core/ExtensionMethods.cs

```diff
@@ -304,7 +304,7 @@ public static string ConvertToUserVariable(this String str, object sender)
         /// <param name="targetVariable">the name of the user-defined variable to override with new value</param>
         public static void StoreInUserVariable(this String str, object sender, string targetVariable)
         {
-            Core.Automation.Commands.VariableCommand newVariableCommand = new Core.Automation.Commands.VariableCommand
+            Core.Automation.Commands.SetVariableCommand newVariableCommand = new Core.Automation.Commands.SetVariableCommand
```

Comment:
I know that SetVariableCommand was originally VariableCommand, but this might have been a bug. I think whoever created this command meant to use NewVariableCommand instead. Also, that's an awful way to create variables anyway. We're getting rid of that.

Reply:
SetVariableCommand is actually meant to modify the value of existing variable but it also creates a variable if it doesn't already exist.

Reply:
Okay, leave it like this for now. We'll modify it later so it's not invoking a command and uses whichever variable modifying method Miguel decides to keep.

Reply:
done